dll/win32/shell32/lang/hu-HU.rc

```diff
@@ -449,15 +449,15 @@ FONT 8, "MS Shell Dlg", 0, 0, 0x0
 BEGIN
     LTEXT "Regisztrált &fájltípusok:", -1, 7, 10, 180, 10
     CONTROL "", IDC_FILETYPES_LISTVIEW, "SysListView32", LVS_REPORT | LVS_SINGLESEL | LVS_SHAREIMAGELISTS | LVS_SHOWSELALWAYS | WS_BORDER | WS_TABSTOP, 7, 20, 249, 99
-    PUSHBUTTON "&Hozzáadás", IDC_FILETYPES_NEW, 140, 123, 55, 14, WS_TABSTOP
+    PUSHBUTTON "&Hozzáadás", IDC_FILETYPES_NEW, 140, 123, 55, 14, WS_TABSTOP //FIXME: accelerator collision &H
     PUSHBUTTON "&Törlés", IDC_FILETYPES_DELETE, 200, 123, 55, 14, WS_TABSTOP
     GROUPBOX "'%s' kiterjesztés részletei", IDC_FILETYPES_DETAILS_GROUPBOX, 7, 143, 249, 70
     LTEXT "Megnyitás ezzel:", -1, 12, 153, 55, 10
     CONTROL "", IDC_FILETYPES_ICON, "STATIC", SS_BITMAP | SS_REALSIZEIMAGE, 85, 153, 10, 10
     LTEXT "", IDC_FILETYPES_APPNAME, 100, 153, 80, 10
     PUSHBUTTON "&Módosítás...", IDC_FILETYPES_CHANGE, 200, 153, 50, 14, WS_TABSTOP
     LTEXT "", IDC_FILETYPES_DESCRIPTION, 12, 168, 160, 40
-    PUSHBUTTON "&Haladó", IDC_FILETYPES_ADVANCED, 200, 188, 50, 14, WS_TABSTOP
+    PUSHBUTTON "&Haladó", IDC_FILETYPES_ADVANCED, 200, 188, 50, 14, WS_TABSTOP //FIXME: accelerator collision &H
```

Comment:
```suggestion
    PUSHBUTTON "Ha&ladó", IDC_FILETYPES_ADVANCED, 200, 188, 50, 14, WS_TABSTOP
```
An example where this accelerator is used:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=Ha%26lad

Otherwise, put it on the first "a" , like here:
https://git.reactos.org/?p=reactos.git&a=search&h=HEAD&st=grep&s=H%26alad